python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -3288,6 +3288,32 @@ def test_as_column_types():
 
     assert_eq(pds, gds)
 
+    pds = pd.Series([])
```

Comment:
Could we also add the specific test cases in #2319, i.e.?

```
print(columnops.as_column(cudf.Series([]), 
print(columnops.as_column(cudf.Series([]), dtype='float32').dtype)
print(columnops.as_column(cudf.Series([]), dtype='str').dtype)
```

Reply:
@shwina Added them into unit-tests 👍